infra/terraform/environments/alpha/main.tf

```diff
@@ -86,6 +86,8 @@ module "container_registry" {
 
 module "data_access" {
```

Comment:
I'm not sure `data_access` is the best name for this module, as that's not really what it's doing - 'data access' is really the shorthand name of our sprint goal. I think `webhook_api` would probably be better?